Makefile

```diff
@@ -201,7 +205,15 @@ release: $(GITHUB_RELEASE)
 	    manifests/cluster-network-addons/cluster-network-addons.package.yaml \
 	    $(shell find manifests/cluster-network-addons/$(shell hack/version.sh) -type f)
 
-vendor: $(GO)
+check-go-version:
+	@if [ "$(CURRENT_GO_VERSION)" != "$(MAX_GO_VERSION)" ]; then \
```

Comment:
It would be great to use -le operator, but I understand since the versions are string, it would be much more complicated to make it work. This is a good way how to tell reviewers that something should be checked.

Reply:
thanks, ye lets start easy please imho

Reply:
basically the trick is to check sort -V, which sort it according semver
just need to take care on all cases where some can be x.y, and other x,y,z etc
so started simply